{{cookiecutter.project_slug}}/templates/blocks/image_block.html

```diff
@@ -1,7 +1,13 @@
 {% load wagtailimages_tags %}
 
 <figure>
-  {% image self.image fill-600x600 %}
+  {% if self.link %}
+    <a href="{{ self.link }}" {% if self.new_window %}target="_blank"{% endif %}>
+      {% image self.image max-1024x768 alt=self.description %}
```

Comment:
One of the repeating things we get from partners is: I've set a focal point, and it does nothing!

Looking at: https://docs.wagtail.io/en/v2.11.3/topics/images.html

Focal point is only used on `fill-` images. By switching to max as default, my instinct is that we'll face this question more often.

Reply:
Hmm, very good point. I'll have a think about this on Monday!

Reply:
I think actually for image blocks this should be fine - the images a partner should be putting into an image block should already be cropped to show the area they're after before being uploaded. The focal point/cropping should only be being used for things like header images, images on cards or in list views. Does that make sense?

Reply:
👍 - let's make sure we remember that knowledge for those sorts of cases in PRs!